wright-filter-wheels/wright_filter_wheels/_yaqd_wright_filter_wheels_continuous.py

```diff
@@ -54,21 +59,21 @@ def direct_serial_write(self, message):
     def home(self):
         loop = asyncio.get_event_loop()
         loop.create_task(self._home())
+        self._set_position(self._position)
```

Comment:
This must be in `_home`, after the wait in order to work as desired, also, it should be self._destination

Reply:
fixed